truss-chains/truss_chains/framework.py

```diff
@@ -720,6 +720,10 @@ def import_target(
 ) -> Iterator[Type[definitions.ABCChainlet]]:
     module_path = pathlib.Path(module_path).resolve()
     module_name = module_path.stem  # Use the file's name as the module name
+    if not os.path.isfile(module_path):
+        raise ImportError(
+            f"`{module_path}` is not a file. You must point to a file where the entrypoint is defined"
```

Comment:
Nit: End all comments and messages with punctuation.

Additional we could enforce here that the path should be a *python source file*, but it's probably clear enough either way...

Reply:
updated with a period! figured it's clear either way, but open to revisiting